src/lsst_efd_client/efd_helper.py

```diff
@@ -671,3 +868,382 @@ def _parse_schema(topic, schema):
             else:
                 vals["is_array"].append(False)
         return pd.DataFrame(vals)
+
+
+class EfdClientSync(_EfdClientInterface, _EfdClientStatic):
+    """Class to handle connections and basic queries synchronously
+
+    Parameters
+    ----------
+    efd_name : `str`
+        Name of the EFD instance for which to retrieve credentials.
+    db_name : `str`, optional
+        Name of the database within influxDB to query ('efd' by default).
+    creds_service : `str`, optional
+        URL to the service to retrieve credentials
+        (``https://roundtable.lsst.codes/segwarides/`` by default).
+    timeout : `int`, optional
+        Timeout in seconds for async requests (`aiohttp.ClientSession`). The
+        default timeout is 900 seconds.
+    client : `object`, optional
+        An instance of a class that ducktypes as
+        `aioinflux.client.InfluxDBClient`. The intent is to be able to
+        substitute a mocked client for testing.
+    """
+
+    mode = ClientMode.SYNC
+
+    def __init__(
+        self,
+        efd_name,
+        db_name="efd",
+        creds_service="https://roundtable.lsst.codes/segwarides/",
+        timeout=900,
+        client=None,
+    ):
+        self._influx_client = EfdClientTools.get_client(
+            efd_name,
+            EfdClientSync.mode,
```

Comment:
This is just the same as using `self.mode` here, right? Is there a reason it's written like this rather than just using it like that?

Reply:
i just use this way to let know is a static attribute, but it can be changed to self